app/assets/javascripts/application.js

```diff
@@ -11,4 +11,5 @@
 // about supported directives.
 //
 //= require rails-ujs
-//= require bootstrap
+//= require card/dist/card
```

Comment:
Move card/dist/card.js to /card and remove all other files under card

Reply:
Done